hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/realtime/RealtimeCompactedRecordReader.java

```diff
@@ -193,6 +195,7 @@ public void close() throws IOException {
     // need clean the tmp file which created by logScanner
     // Otherwise, for resident process such as presto, the /tmp directory will overflow
     ((ExternalSpillableMap) deltaRecordMap).close();
+    mergedLogRecordScanner.close();
```

Comment:
minor comment：  maybe we can remove line197， since mergedLogRecordScanner.close() will call ((ExternalSpillableMap) deltaRecordMap).close() internal